### PR TITLE
Necessary permissions for pipeline Service Account

### PR DESCRIPTION
**Pull request by MarcinGinszt**
Service Account `pipeline` used by openshift pipelines needs permissions for project creation. They are used in task imagestream-manipulation.

### CODE REVIEW

File: ansible/init.sh [hunk 1]
```diff
@@ -62,6 +62,15 @@ main() {
         execute_playbook $env $passwd_file
     done
 
+    # Grant the pipeline service account permissions to create projects
+    # -it cannot be done in Playbook, as ansible SA has no permissions to create
+    # cluster-scoped resources- and this command creates ClusterRoleBinding
+    for env in $environments;
+    do
+        oc adm policy add-cluster-role-to-user self-provisioner -z pipeline -n operator-pipeline-$env
```
Comment (Allda) [inline ansible/init.sh:70]: Can you add the oc command the the loop above? I don't see any benefit of duplicating a code.

File: ansible/init.sh [hunk 2]
```diff
@@ -62,6 +62,15 @@ main() {
         execute_playbook $env $passwd_file
     done
 
+    # Grant the pipeline service account permissions to create projects
```
Comment (Allda) [inline ansible/init.sh:65]: This step requires a pipelines operator to be installed on a cluster. Can you please mention it somewhere in a doc? Or maybe even better install the pipeline operator in an ansible role.

Comment (MarcinGinszt) [reply ansible/init.sh]: Most of the steps (eg. installing tasks and pipelines) already required the pipeline operator. Can I create an additional ticket to do that?

Comment (MarcinGinszt) [reply ansible/init.sh]: https://issues.redhat.com/browse/ISV-1115

**Review by Allda — APPROVED** [review]

**Pull request merged**